config/locales/client.en.yml

```diff
@@ -3431,6 +3434,15 @@ en:
         lower_title_with_count:
           one: "%{count} unread"
           other: "%{count} unread"
+      unseen:
+        title: "Unseen"
+        title_with_count:
```

Comment:
Based on our internal discussion, we've decided to not display a count for this view.

Reply:
I've fixed it, thank you!